src/main/resources/lib/credentials/select.jelly

```diff
@@ -75,6 +80,27 @@
     </j:if>
     <div field-disabled="${paramValue?'true':null}"
          class="credentials-select-content-select ${paramValue?'credentials-select-content-inactive':'credentials-select-content-active'}">
+      <j:choose>
+        <j:when test="${includeUser and attrs.includeUserCaution}">
+          <!-- TODO: update the build-with-parameters.png screenshot in documentation -->
```

Comment:
Does this TODO need to be completed?

Reply:
Ideally, yes. I need to set up a test project to take a similar screenshot to the current one.